xero-app-store.yaml

```diff
@@ -299,3 +458,116 @@ components:
         usageUnit:
           description: The unit of the usage product. e.g. "user", "minutes", "SMS", etc
           type: string
+    UsageRecord:
+      properties:
+        quantity:
+          description: The quantity recorded
+          format: int32
+          type: integer
+        subscriptionId:
+          description: The unique identifier of the Subscription.
+          format: guid
+          type: string
+        subscriptionItemId:
+          description: The unique identifier of the SubscriptionItem.
+          format: guid
+          type: string
+        testMode:
+          description: If the subscription is a test subscription
+          type: boolean
+        timeStamp:
+          description: The timestamp when this usage occurred in UTC
+          format: date-time
+          type: string
+        usageRecordId:
+          description: The unique identifier of the usageRecord.
+          format: guid
+          type: string
+      required:
+        - usageRecordId
+        - subscriptionId
+        - subscriptionItemId
+        - quantity
+        - testMode
+        - timeStamp
+      type: object
+    UsageRecordsListItem:
+      properties:
+        pricePerUnit:
+          description: The price per unit
+          format: decimal
+          type: number
+        productId:
+          description: The unique identifier of the linked Product
+          format: guid
+          type: string
+        quantity:
+          description: The quantity recorded
+          format: int32
+          type: integer
+        subscriptionId:
+          description: The unique identifier of the linked Subscription
+          format: guid
+          type: string
+        subscriptionItemId:
+          description: The unique identifier of the linked Subscription Item
+          format: guid
+          type: string
+        testMode:
+          description: If the subscription is a test subscription
+          type: boolean
+        timeStamp:
+          description: The timestamp when this usage occurred in UTC
+          format: int64
```

Comment:
Based on the example this should be `format: date-time  type: string`

Reply:
TY, I've removed this `UsageRecordsListItem` - it's now referencing the parent list now references `UsageRecord`